Add General Notes (Public) field to Crew Records

SierraKomodo opened the pull request:
Adds a 'General Record' field to crew records. This serves as a field that anyone with crew records access can see, for people (Like me) that want to add 'common' information between medical, employment, and security records.

General records can be edited in game by anyone with access to the bridge (bridge officers and any head of staff)

![2018-11-26_14-33-54](https://user-images.githubusercontent.com/11140088/49046338-a459e080-f188-11e8-9397-830d36fb1201.png)

:cl: SierraKomodo
add: General Notes (Public) field has been added to crew records. These are records visible to anyone who can open crew records, and are intended for 'common' information between security, medical, /and/ employment records.
/:cl:

SierraKomodo committed:
Add General Record field

afterthought2 commented:
How about calling it `Notes` or something? General record is confusing as to what goes in it, and the example in the screenshot isn't very good because a lot of it is redundant with the autofilled short fields above it.

SierraKomodo commented:
`Notes` could work, though I'd be a bit concerned with that being confused with IC Notes, and the name also not conveying what the intent of this field is.

SierraKomodo commented:
Changed the field name in the UI and the variable names to better fit/match, and updated the example screenshot with a better, less repetitive example.

comma reviewed (approved):
CItizenship field exists too (only visible to those who have employment records) but overall better
